src/components/TagDescriptionEditor.js

```diff
@@ -34,7 +63,10 @@ export default class TagDescriptionEditor extends React.Component {
         {map(tags, (description, tag) => <div key={tag} className='tag-group'>
           <div className='topic'>
             <label>Topic</label>
-            <span>#{tag}</span>
+            {creating
+            ? <span>#&nbsp;<input type='text' defaultValue={tag} className='tag'
```

Comment:
move the <input> to its own line -- there's an implicit style rule that we've been consistently following, which is that if a tag is on the same line as its parent's start tag, then the parent's end tag must also be on that same line